setup.cfg

```diff
@@ -18,7 +18,7 @@ classifiers =
     Programming Language :: Python :: 3.10
 
 [bumpversion]
-current_version = 0.2.0
+current_version = 0.3.0-dev
```

Comment:
📓: When we actually create a tag, the tag would be this version and then this would have to be bumped up again